src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -247,15 +247,15 @@ private void validateAttachments(final Attachable attachable) throws GameParseEx
 
   public Document getDocument(final InputStream input) throws SAXException, IOException, ParserConfigurationException {
     final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-    factory.setValidating(true);
-    // get the dtd location
-    final String dtdFile = "/games/strategy/engine/xml/" + DTD_FILE_NAME;
-    final URL url = GameParser.class.getResource(dtdFile);
-    if (url == null) {
-      throw new RuntimeException("Map: " + mapName + ", " + String.format("Could not find in classpath %s", dtdFile));
-    }
-    final String dtdSystem = url.toExternalForm();
-    final String system = dtdSystem.substring(0, dtdSystem.length() - 8);
```

Comment:
It's only used in test. That is not helpful since the real ones can be arbitrary.